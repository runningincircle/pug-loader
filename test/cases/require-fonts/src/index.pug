html
  head
    link(rel="preload" href=require('./assets/fonts/ibm-plex-sans-v7-latin-regular.woff2') as="font" type="font/woff2" crossorigin)
  body
    p Hello World!
    img(src=require('./image.png'))