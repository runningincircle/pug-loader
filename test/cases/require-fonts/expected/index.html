<!DOCTYPE html><html><head><link rel="preload" href="/assets/fonts/ibm-plex-sans-v7-latin-regular.woff2" as="font" type="font/woff2" crossorigin></head><body><p>Hello World!</p><img src="/assets/images/image.697ef306.png"></body></html>